presto-main/src/main/java/com/facebook/presto/tracing/testing/TestingPoint.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tracing.testing;
+
+public class TestingPoint
+{
+    private String annotation;
+    private long time;
+
+    public TestingPoint(String annotation)
+    {
+        this.annotation = annotation;
+        this.time = System.currentTimeMillis();
+    }
+
+    public String getAnnotation()
+    {
+        return annotation;
+    }
+
+    public long getTime()
+    {
+        return time;
+    }
+
+    public String toString()
+    {
+        return "{\n" +
+                "  \"annotation\": \"" + annotation + "\",\n" +
+                "  \"time\": " + time + "\n" +
+                "}";
+    }
```

Comment:
same, remove?

Reply:
it will be useful to get the overall tracing structure.